SamsungPay/src/main/java/com/braintreepayments/api/SamsungPayPartnerInfoBuilder.java

```diff
@@ -0,0 +1,71 @@
+package com.braintreepayments.api;
+
+import android.os.Bundle;
+
+import com.samsung.android.sdk.samsungpay.v2.PartnerInfo;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import static com.samsung.android.sdk.samsungpay.v2.SpaySdk.PARTNER_SERVICE_TYPE;
+import static com.samsung.android.sdk.samsungpay.v2.SpaySdk.ServiceType.INAPP_PAYMENT;
+import static com.samsung.android.sdk.samsungpay.v2.payment.PaymentManager.EXTRA_KEY_TEST_MODE;
+
+class SamsungPayPartnerInfoBuilder {
+
+    private static final String API_VERSION_KEY = "braintreeTokenizationApiVersion";
+    private static final String CLIENT_SDK_METADATA_KEY = "clientSdkMetadata";
+
+    private static final String API_VERSION = "2018-10-01";
```

Comment:
Is this accurate? Is there a newer api version we should be upgrading to?

Reply:
Yeah but we can rename to `BRAINTREE_API_VERSION` I think to be more clear.